### PR TITLE
CHANGES: add release notes for v6.10.0

### PR DESCRIPTION
**Pull request by evan-charmworks**
*Original date: 2019-05-08 04:12:33*
*Original PR: https://charm.cs.illinois.edu/gerrit/5152*

---

Change-Id: Iee9fd0da868856c0f6461800fcdd0b99a1c4523e

### CODE REVIEW

**Comment by stwhite91** [comment]
Feel free to edit this PR for any features/fixes I may have missed

File: CHANGES [hunk 1]
```diff
@@ -1,6 +1,131 @@
...
+
+- We have moved away from .tex in favor of .rst files to make building the
+  documentation more portable. The documentation is now available at
+  https://charm.readthedocs.io/ .
+
+- We have moved bug/issue tracking from Redmine to GitHub, and code
+  review from Gerrit to GitHub. Our GitHub repository is at:
+  https://github.com/UIUC-PPL/charm .
+  
+- As a preview feature, Charm++ can now be build with CMake (version 3.11 or higher).
```
Comment (rbuch) [inline CHANGES]:
```suggestion
- As a preview feature, Charm++ can now be built with CMake (version 3.11 or higher).
```

**Comment by stwhite91** [comment]
Looks to be ready for final review / merge now

File: CHANGES [hunk 2]
```diff
@@ -36,6 +36,7 @@ Known Issues:
 
 - A hang occurs in multiprocess runs in SMP mode with completion detection on GNI builds.
   (https://github.com/UIUC-PPL/charm/issues/2508)
+
 - Recent InfiniBand machines crash in SMP builds due to problem in verbs layer implementation.
```
Comment (matthiasdiener) [inline CHANGES]: a problem/problems?

File: CHANGES [hunk 3]
```diff
@@ -1,6 +1,148 @@
...
+  infrastructure. Programs that rely on custom load balancers or the internals of the LB
+  infrastructure will likely require some changes for compatibility.
+  
+Known Issues:
+
+- A hang occurs in multiprocess runs in SMP mode with completion detection on GNI builds.
+  (https://github.com/UIUC-PPL/charm/issues/2508)
+
+- Recent InfiniBand machines crash in SMP builds due to problems in the verbs layer implementation.
+  Users are recommended to use UCX in the time being if possible.
```
Comment (matthiasdiener) [inline CHANGES]: maybe: for the time being

**Pull request merged**